components/blocks/Landing/LandingLeaderboards.test.ts

```diff
@@ -0,0 +1,31 @@
+import { mountSuspended } from '@nuxt/test-utils/runtime'
+import Landing from './LandingLeaderboards.vue'
+import { useGetLeaderboards } from '~/composables/api'
+
+const mockSuccessGetLeaderboards = vi.fn(() => Promise.resolve({ ok: true }))
+vi.mock('lib/api/Leaderboards', () => ({
+  Leaderboards: function Leaderboards() {
+    this.getLeaderboards = mockSuccessGetLeaderboards
+  },
+}))
+
+const leaderboards = await useGetLeaderboards()
```

Comment:
I don't think it makes sense to mock this import and then immediately call it. Just pass the test data into the component directly.